methods_test.test.js

```diff
@@ -1,42 +1,74 @@
-// import capitalize from './methods_to_test';
-const { capitalize, reverseString, analyse, Calculator, Cypher } = require('./methods_to_test.js')
 
+const { capitalize, reverseString, analyse, Calculator, Cypher } = require('./methods_to_test.js')
 
-test("capitalize", () => {
-    expect(capitalize("name")).toMatch(/[N]ame/)
+test.each([
+    ['name', /[N]ame/], 
+    ['Jason', /[J]ason/]
+])('capitalize', (name, result) => {
+    expect(capitalize(name)).toMatch(result)
 })
 
-test("reverse string", () => {
-    expect(reverseString("string")).toMatch(/gnirts/)
+test.each([
```

Comment:
We want our tests to be easily understood and very readable.

Let us write our tests this way:

```javascript
describe('capitalize function, () => {
  it("should capitalize 'name''", () => {
    expect(capitalize('name').toBe('Name');
  }
  it("should capitalize 'jason'", ()=> {
    expect(capitalize('jason').toBe('Jason');
  })
  // ... at least one more test case
});

describe('reverseString function', ()=> {
  // same format as in the tests for 'capitalize'
})
```